### ISSUE TITLE
An error has occurred: Could not find Windows Runtime type 'Windows.Media.SystemMediaTransportControls'..

### ISSUE THREAD

**Issue by alexeisolonari** [Royce551/FRESHMusicPlayer#3]
Error occurred on Windows 7 on first open. 
![Error window](https://user-images.githubusercontent.com/17863082/98598576-b4512900-22a8-11eb-822c-fd53130cefeb.PNG)

Log file:
```
System.TypeLoadException: Could not find Windows Runtime type 'Windows.Media.SystemMediaTransportControls'. ---> System.PlatformNotSupportedException: Operation is not supported on this platform.
   --- End of inner exception stack trace ---
   at FRESHMusicPlayer.MainWindow.UpdateIntegrations()
   at FRESHMusicPlayer.MainWindow.<Window_SourceInitialized>d__13.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by Royce551** [comment]
apparently doing anything with winrt types just doesn't work in windows 7, should be able to fix for fmp 9.4 or 10

**Comment by Royce551** [comment]
fixed in fmp 10 (i think)
